src/frontend/js/widgets/Dashboard/components/DashboardItem/index.tsx

```diff
@@ -66,3 +81,28 @@ export const DashboardItem = ({
     </div>
   );
 };
+
+const DashboardItemMore = ({ more }: { more: ReactNode }) => {
+  const intl = useIntl();
+  const { isOpen, getToggleButtonProps, getMenuProps } = useSelect({
+    items: [],
+  });
+
+  return (
+    <div className="selector dashboard-item__block__head__more">
```

Comment:
If you make it reusable, `dashboard-item__block__head__more` should only handle menu position in `DashboardItem` context.